build.yaml

```diff
@@ -24,4 +24,14 @@ build:
   - npm: install
   - npm: run-script ci
     graceful: true
+  - script: |
+      cd examples
+      npm install
+      npm install ../
```

Comment:
The `npm install ../` step points to the local cassandra driver, where as if i were to just use npm install, it'd use the version from master.